README.md

```diff
@@ -132,10 +132,15 @@ Currently the different available records are
 
 ``` js
 {
-  data: Buffer('some text')
+  data: Buffer(...)
+}
+{
+  data: 'some string'
 }
 ```
 
+Note that `data` must be an array of byte-length prefixed strings and the end of the array is signalled by a length of zero.  When encoding, a Buffer is taken as is (assumption is that it is encoded correctly), while a string is converted.
```

Comment:
I don't understand this sentence. What is a "byte-length prefixed string"?

Also, what should happen if the user introduces null-bytes in the string, should be throw?

Reply:
Removed, don't need to talk about internal encoding any more.

Reply:
NUL bytes are okay.